android/src/main/java/com/brentvatne/exoplayer/ReactExoplayerView.java

```diff
@@ -626,8 +647,10 @@ private void initializePlayerCore(ReactExoplayerView self) {
         );
         DefaultRenderersFactory renderersFactory =
                 new DefaultRenderersFactory(getContext())
+                        .setEnableDecoderFallback(true)
```

Comment:
I think this can lead to performancies issue, can you disable it if user uses SurfaceView ?
But maybe also if frameQuality is null ? No sure this is feasible because you will need to change the value if user set dynamically frameQuality.

Reply:
no effect if surfaceview is used